plugin/src/main/java/com/denizenscript/denizen/events/entity/ExperienceBottleBreaksScriptEvent.java

```diff
@@ -0,0 +1,103 @@
+package com.denizenscript.denizen.events.entity;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.Argument;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.ExpBottleEvent;
+
+public class ExperienceBottleBreaksScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // experience bottle breaks
+    //
+    // @Regex ^on experience bottle breaks$
+    //
+    // @Switch in <area>
+    //
+    // @Cancellable true
+    //
+    // @Triggers when a thrown experience bottle breaks.
+    //
+    // @Context
+    // <context.entity> returns the EntityTag of the thrown experience bottle.
+    // <context.experience> returns the amount of experience to be spawned.
+    // <context.show_effect> returns whether the effect should be shown.
+    //
+    // @Determine
+    // "NO_EXPERIENCE" to specify that no experience orbs should be dropped.
+    // "NO_EFFECT" to specify that particles should be shown.

```

Comment:
the `no_` prefix determination options are redundant and pointless if they're directly equivalent to other determination options